packages/blueprints-integration/src/config.ts

```diff
@@ -109,6 +113,19 @@ export interface ConfigManifestEntrySelectFromColumn<Multiple extends boolean>
 	columnId: string
 }
 
+export type ComparisonMapping = { targetColumnId: string; sourceColumnId: string }
+
+export interface ConfigManifestEntrySelectFromTableEntryWithComparisonMappings<Multiple extends boolean>
+	extends ConfigManifestEntrySelectBase<Multiple> {
+	type: ConfigManifestEntryType.SELECT_FROM_TABLE_ENTRY_WITH_COMPARISON_MAPPINGS
+
+	sourceTableId: string
+
+	comparisonMappings: ComparisonMapping[]
+
+	sourceColumnIdWithValue: string
```

Comment:
Is it necessary to specify `...WithValue`? It's the column it has to target no matter if it has been populated or not?

Reply:
You have to specify three different columns, the one you mention and two (or more potentially) in the `comparisonMappings`. The idea is not to suggest it is populated, but to indicate that this is where values are gathered from. If you have a better suggestion i am very much open to it!